docs/content/installation/integrations/app-protect-waf-v5/installation.md

```diff
@@ -0,0 +1,319 @@
+
+To do so, place the `*.crt` file in the build folder and uncomment the lines following this comment:
+`#Uncomment the lines below if you want to install a custom CA certificate`
+
+{{<warning>}} External references are deprecated in NGINX Ingress Controller and will not be supported in future releases. {{</warning>}}
+
+---
+
+## Push the images to your private registry
+
+Once you've successfully pulled the WAF v5 manager and enforcer images and built the NGINX Ingress Controller image with NGINX App Protect WAF v5, the next step is to upload them to your private Docker registry. This makes the image available for deployment to your Kubernetes cluster.
+
+To upload the image, run the following command. If you're using a custom tag, add `TAG=your-tag` to the end of the command. Replace `<my-docker-registry>` with your private registry's path.
+
+```shell
+make push PREFIX=<my-docker-registry>/nginx-plus-ingress
+```
+
+To upload the WAF config manager and enforcer images run the following commands:
+
+```shell
+docker push <my-docker-registry>/waf-config-mgr:<your-tag>
+```
+
+```shell
+docker push <my-docker-registry>/waf-enforcer:<your-tag>
+```
+
+---
+
+## Set up role-based access control (RBAC) {#set-up-rbac}
+
+{{< include "rbac/set-up-rbac.md" >}}
+
+---
+
+
+{{< include "installation/create-custom-resources.md" >}}
+
+
+## Deploy NGINX Ingress Controller {#deploy-ingress-controller}
+
+{{< important >}} NGINX Ingress Controller with the AppProtect WAF v5 module works only with policy bundles. You need to modify the Deployment or DaemonSet file to include volumes, volume mounts and two WAF 5 docker images: `waf-config-mgr` and `waf-enforcer`.
+
+NGINX Ingress Controller **requires** the volume mount path to be `/etc/app_protect/bundles`. {{< /important >}}
+
+{{<tabs name="deploy-nic">}}
+
+{{%tab name="With Helm"%}}
+
+Below are examples of a `PersistentVolume` and `PersistentVolumeClaim` that you can reference in your Helm values:
+
+```yaml
+...
+volumes:
+- name: <volume_name>
+persistentVolumeClaim:
+    claimName: <claim_name>
+...
+```
+
+Add volume mounts to the `containers` section:
+
+```yaml
+...
+volumeMounts:
+- name: <volume_mount_name>
+    mountPath: /etc/app_protect/bundles
+...
+```
+
+### Enabling WAF v5
+
+Start by setting `controller.appprotect.enable` to `true` in your Helm values. This will the standard App Protect WAF fetatures.
+Afterwords, set `controller.approtect.v5` to `true`.
+This ensures that both the `waf-enforcer` and `waf-config-mgr` containers are deployed alongside the NGINX Ingress Controller containers.
+These two additional containers are required when using App Protect WAF v5.
+
+Your Helm values should look something like this:
+
+```yaml
+controller:
+  ...
+  ## Support for App Protect WAF
+  appprotect:
+    ## Enable the App Protect WAF module in the Ingress Controller.
+    enable: true
+    ## Enables App Protect WAF v5.
+    v5: true
+```
+
+
+### Configuring volumes
```

Comment:
It does also work with emptyDirs, even if its not scalable. Maybe add a point like:
> By default emptyDirs mounts will be used, but bundles added to these will not persist across pod restarts.